packages/typescript-estree/tests/lib/parse.test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Noting for self: this file has an `expect.any(Array)` that's firing triggering the rule... how can we work around that?

Reply:
I've played around with this in other repos and most of the time ended up disabling `@typescript-eslint/no-unsafe-assignment` in tests altogether. `expect`, `jest.fn`, and other test utilities often use `any` in their types unfortunately.

Reply:
e5496f72